products/green_fast_track/seeds/_seeds.yml

```diff
@@ -15,8 +15,6 @@ seeds:
         tests:
           - not_null
       - name: survey_question
-        tests:
-          - not_null
```

Comment:
Columns seem a tad in flux, and we don't use them from this variable column so made them optional for now